.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,16 @@
+# Contributing to VTEX Styleguide
+
+Want to contribute to VTEX Styleguide? There are a few things you need to know.
+
+### Release Schedule
+Thursday release: patch version at the thursday of every week for routine bugfix (anytime for urgent bugfix).
+
+Tuesday release: minor version at the tuesday of every month for new features.
+
+Major version release is not included in this schedule for breaking change and new features.
```

Comment:
```suggestion
Major version release with possible breaking-changes is not included in this schedule.
```

Reply:
I'm not not the best fan of my change also, but the original is kind of weird too 🤔

Reply:
@klzns' suggestion was more pleasant.

Reply:
anyway thank you!